Lib/unittest/case.py

```diff
@@ -149,9 +149,9 @@ def doModuleCleanups():
         except Exception as exc:
             exceptions.append(exc)
     if exceptions:
-        # Swallows all but first exception. If a multi-exception handler
-        # gets written we should use that here instead.
-        raise exceptions[0]
+        if len(exceptions) == 1 and not isinstance(exceptions[0], ExceptionGroup):
+            raise exceptions[0]
+        raise ExceptionGroup('module cleanup failed', exceptions)
```

Comment:
I think this should consistently raise `ExceptionGroup('module cleanup failed', exceptions)`, even when `len(exceptions) == 1`.

Reply:
Done. But this changed outcome of some existing tests.